dev/io.openliberty.webcontainer.servlet.6.1.internal_fat/publish/servers/servlet61_PathCanonicalizationTest/server.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+    Copyright (c) 2024 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+ -->
+<server description="Server for testing path canonicalization for ServletContext methods what accept path parameter">
+    <include location="../fatTestPorts.xml"/>
+
+    <featureManager>
+        <feature>servlet-6.1</feature>
+    </featureManager>
+    
+        <webContainer com.ibm.ws.webcontainer.getrealpathreturnsqualifiedpath="false"/>  
```

Comment:
This property's default needs to be flipped.  Doc it in the TODO issue
https://github.com/OpenLiberty/open-liberty/issues/27383

Reply:
@pmd1nh Can you expand on this a bit? This doesn't seem to be something that has changed in Servlet 6.1 is it? Have you tested any other servers to see what the behavior is? We made this change a while ago to have some consistency between tWAS and Open Liberty.

See the Servlet 4.0 specification for example https://jakarta.ee/specifications/servlet/4.0/apidocs/javax/servlet/servletcontext#getRealPath-java.lang.String-

Why do we need it set to false for this test case?

Reply:
With the default `true` value of the property, getRealPath of a bogus path returns

testPath BOGUS PATH with DEFAULT property [/bogus/path/ImNotExist.html] ; return [/Users/pmdinh/1_WAS/ASLiberty/2023_Nov/open-liberty/dev/build.image/wlp/usr/servers/servlet61_PathCanonicalizationTest`/apps/expanded/PathCanonicalizationTest.war/bogus/path/ImNotExist.html`]

With property set to `false`

testPath BOGUS PATH with property = FALSE [/bogus/path/ImNotExist.html] ; return [null]

Return a `real` path for a `bogus` virtual path does not look right to me.
@pnicolucci

Reply:
Phu and I discussed this and we're not changing this for Servlet 6.1 the getRealPath here should return a value where the file would have been if it did exist rather than null. We're behaving as we expect here.